deploy/host-operator/e2e-tests/toolchainconfig.yaml

```diff
@@ -3,11 +3,34 @@ kind: ToolchainConfig
 metadata:
   name: config
 spec:
+  host:
+    deactivation:
+      deactivationDomainsExcluded: '@excluded.com'
+    environment: 'e2e-tests'
+    notifications:
+      durationBeforeNotificationDeletion: '5s'
+    registrationService:
+      analytics:
+        segmentWriteKey: 'test segment write key'
+        woopraDomain: 'test woopra domain'
+      environment: 'e2e-tests'
+      verification:
+        enabled: true
+        excludedEmailDomains: 'redhat.com,acme.com'
+        secret:
+          ref: 'host-operator-secret'
+          twilioAccountSID: 'twilio.account.sid'
+          twilioAuthToken: 'twilio.auth.token'
+          twilioFromNumber: 'twilio.from_number'
+    tiers:
+      durationBeforeChangeTierRequestDeletion: '5s'
+    toolchainStatus:
+      toolchainStatusRefreshTime: '1s'
```

Comment:
I see that you don't specify the number of replicas of the reg-service as we do in `e2e.yaml` & `dev.yaml` config files. Wouldn't it be better to drop it from the file as well to make the config consistent? In that case you would also need to modify the e2e test: https://github.com/codeready-toolchain/toolchain-e2e/blob/master/testsupport/init.go#L53

Reply:
Makes sense, I've added replicas in https://github.com/codeready-toolchain/api/pull/253 and I'll update this to include replicas.

Reply:
Actually, I'll add it in a separate PR since it won't be used now anyway. It'll be used in the host operator config migration PR.

Reply:
Added in https://github.com/codeready-toolchain/toolchain-e2e/pull/368